spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -172,20 +172,20 @@ def main(argv=None):
                                                          verbose=verbose)
 
     # save centerline as nifti (discrete) and csv (continuous) files
-    im_centerline.save(file_output + '.nii.gz')
+    im_centerline.save(file_output)
     np.savetxt(file_output + '.csv', arr_centerline.transpose(), delimiter=",")
 
-    display_viewer_syntax([fname_input_data, file_output + '.nii.gz'], colormaps=['gray', 'red'], opacities=['', '1'])
+    display_viewer_syntax([fname_input_data, file_output], colormaps=['gray', 'red'], opacities=['', '1'])
 
     path_qc = arguments.qc
     qc_dataset = arguments.qc_dataset
     qc_subject = arguments.qc_subject
 
     # Generate QC report
     if path_qc is not None:
-        generate_qc(fname_input_data, fname_seg=file_output + '.nii.gz', args=sys.argv[1:], path_qc=os.path.abspath(path_qc),
+        generate_qc(fname_input_data, fname_seg=file_output, args=sys.argv[1:], path_qc=os.path.abspath(path_qc),
                     dataset=qc_dataset, subject=qc_subject, process='sct_get_centerline')
-    display_viewer_syntax([fname_input_data, file_output + '.nii.gz'], colormaps=['gray', 'red'], opacities=['', '0.7'])
+    display_viewer_syntax([fname_input_data, file_output], colormaps=['gray', 'red'], opacities=['', '0.7'])
```

Comment:
Looking at the changes again, I'm realizing that we're actually duplicating our call to `display_viewer_syntax` unnecessarily. 

Now is probably a good time to remove this, aha. :sweat_smile:

Reply:
Done in a6c8728.